### PR TITLE
Danb/mainnet announcement

### PR DESCRIPTION
**Pull request by danbryan**
Initial mainnet announcement, to start collecting gentx files for noble.

### CODE REVIEW

File: mainnet/noble-1/README.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+```
+## Persistent Peers
+```
+#Strangelove
+TBD
+```
+
+## Binary
+
+Docker images are available [here](https://github.com/strangelove-ventures/noble/pkgs/container/noble/72469688?tag=v1.0.0). You can generate the binary by building from the Official Repo. Or alternatively you can use the Verify process below to build inside docker and guarantee you have the correct source.
```
Comment (boojamya) [inline mainnet/noble-1/README.md:45]: This should be the URL:
https://github.com/strangelove-ventures/noble/pkgs/container/noble/78874895?tag=v1.0.0

File: mainnet/noble-1/README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+cd noble
+git checkout v1.0.0
+make install
+```
+### Verify Binary Checksum
+Binary checksums can differ based on many things to include go, libc, and make versions. To get a consistent checksum you can use something like docker to verify.
+
+  * [Linux amd64 build](nobled)
+  * Version: `v1.0.0`
+  * SHA256: `fc92d3b64b0618238b354bd82b31ed4aa35557129d679d90b070643df34c7c38`
```
Comment (boojamya) [inline mainnet/noble-1/README.md:58]: Wouldn't it be?

2704d751d417c7c75907f8c0ae92009d29dda5bc809cb2d7e66b58218479ede3

File: mainnet/noble-1/README.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+  Example of using a volume mount to get the binary outside of the container onto your ubuntu server.
+  ```
+  #run on your ubuntu server
+  # use the `realpath` for the volume mount.
+  docker run -v /home/ubuntu/go/bin:/root/go/bin -it --entrypoint /bin/bash ghcr.io/strangelove-ventures/checksum:v.0.1.0
```
Comment (boojamya) [inline mainnet/noble-1/README.md:64]: Is this supposed to be `v1.0.0`?

File: mainnet/noble-1/README.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+  ```
+  # run inside docker container.
+  git clone https://github.com/strangelove-ventures/noble.git
+  cd noble
+  git fetch
+  git checkout v1.0.0
+  make install
+  sha256sum ~/go/bin/nobled
+  ```
+  expected return `fc92d3b64b0618238b354bd82b31ed4aa35557129d679d90b070643df34c7c38`
```
Comment (boojamya) [inline mainnet/noble-1/README.md:75]: Same as sha noted above?

File: mainnet/noble-1/README.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+  sha256sum ~/go/bin/nobled
+  ```
+  expected return `fc92d3b64b0618238b354bd82b31ed4aa35557129d679d90b070643df34c7c38`  
+  
+  Now, verify the checksum on your local ubuntu server  
+  ```
+  #run on your ubuntu server
+  sha256sum /home/ubuntu/go/bin/nobled
+  ```
+  expected return `fc92d3b64b0618238b354bd82b31ed4aa35557129d679d90b070643df34c7c38`
```
Comment (boojamya) [inline mainnet/noble-1/README.md:82]: Here too?

File: mainnet/noble-1/README.md (hunk 1 above)
Comment (danbryan) [reply mainnet/noble-1/README.md:45]: good catch, wonder where that random number comes from

File: mainnet/noble-1/README.md (hunk 3 above)
Comment (danbryan) [reply mainnet/noble-1/README.md:64]: nope, this is for the checksum image, not noble

File: mainnet/noble-1/README.md (hunk 2 above)
Comment (danbryan) [reply mainnet/noble-1/README.md:58]: Thats not what I get.

```
 ╭─danb@arch in ~  sl-test-val as 🧙 took 3s
 ╰─λ docker run -v /home/ubuntu/go/bin:/root/go/bin -it --entrypoint /bin/bash ghcr.io/strangelove-ventures/checksum:v.0.1.0 
root@f6ef59262f22:/#   git clone https://github.com/strangelove-ventures/noble.git
  cd noble
  git fetch
  git checkout v1.0.0
  make install
  sha256sum ~/go/bin/nobled
Cloning into 'noble'...
remote: Enumerating objects: 3479, done.
remote: Counting objects: 100% (418/418), done.
remote: Compressing objects: 100% (120/120), done.
remote: Total 3479 (delta 304), reused 383 (delta 289), pack-reused 3061
Receiving objects: 100% (3479/3479), 1.78 MiB | 13.19 MiB/s, done.
Resolving deltas: 100% (2453/2453), done.
Note: switching to 'v1.0.0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at c252a78 Bump golang.org/x/net from 0.5.0 to 0.7.0 in /interchaintest (#112)
go install -mod=readonly -tags "netgo ledger" -ldflags '-X github.com/cosmos/cosmos-sdk/version.Name=noble -X github.com/cosmos/cosmos-sdk/version.AppName=nobled -X github.com/cosmos/cosmos-sdk/version.Version=HEAD-c252a78d36944be21a1e847c6781e10aa43f1066 -X github.com/cosmos/cosmos-sdk/version.Commit=c252a78d36944be21a1e847c6781e10aa43f1066 -X "github.com/cosmos/cosmos-sdk/version.BuildTags=netgo ledger," -X github.com/tendermint/tendermint/version.TMCoreSemVer=v0.34.26' ./cmd/nobled
go: downloading github.com/cosmos/cosmos-sdk v0.45.13
go: downloading github.com/spf13/cast v1.5.0
go: downloading github.com/spf13/cobra v1.6.1
go: downloading github.com/informalsystems/tendermint v0.34.26
go: downloading github.com/spf13/pflag v1.0.5
go: downloading github.com/tendermint/tm-db v0.6.7
go: downloading github.com/cosmos/ibc-go/v3 v3.4.0
go: downloading github.com/strangelove-ventures/packet-forward-middleware/v3 v3.1.4
go: downloading github.com/strangelove-ventures/paramauthority v0.1.1-0.20230120184403-39a8bf5f845d
go: downloading github.com/rs/zerolog v1.27.0
go: downloading github.com/regen-network/protobuf v1.3.3-alpha.regen.1
go: downloading github.com/hashicorp/golang-lru v0.5.5-0.20210104140557-80c98217689d
go: downloading github.com/pkg/errors v0.9.1
go: downloading golang.org/x/exp v0.0.0-20220722155223-a9213eeb770e
go: downloading gopkg.in/yaml.v2 v2.4.0
go: downloading github.com/gorilla/mux v1.8.0
go: downloading github.com/grpc-ecosystem/grpc-gateway v1.16.0
go: downloading github.com/spf13/viper v1.14.0
go: downloading github.com/rakyll/statik v0.1.7
go: downloading github.com/stretchr/testify v1.8.1
go: downloading github.com/golang/protobuf v1.5.2
go: downloading google.golang.org/grpc v1.33.2
go: downloading google.golang.org/protobuf v1.28.2-0.20220831092852-f930b1dc76e8
go: downloading github.com/cosmos/go-bip39 v1.0.0
go: downloading github.com/regen-network/cosmos-proto v0.3.1
go: downloading google.golang.org/genproto v0.0.0-20230113154510-dbe35b8444a5
go: downloading github.com/tendermint/go-amino v0.16.0
go: downloading github.com/grpc-ecosystem/go-grpc-middleware v1.3.0
go: downloading github.com/confio/ics23/go v0.9.0
go: downloading github.com/cosmos/btcutil v1.0.4
go: downloading github.com/gogo/gateway v1.1.0
go: downloading github.com/gorilla/handlers v1.5.1
go: downloading github.com/armon/go-metrics v0.4.0
go: downloading github.com/99designs/keyring v1.2.1
go: downloading cosmossdk.io/errors v1.0.0-beta.7
go: downloading github.com/google/btree v1.0.1
go: downloading github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
go: downloading github.com/iancoleman/orderedmap v0.2.0
go: downloading github.com/mattn/go-colorable v0.1.13
go: downloading github.com/hdevalence/ed25519consensus v0.0.0-20220222234857-c00d1f31bab3
go: downloading github.com/btcsuite/btcd v0.22.2
go: downloading golang.org/x/crypto v0.5.0
go: downloading github.com/fsnotify/fsnotify v1.6.0
go: downloading github.com/mitchellh/mapstructure v1.5.0
go: downloading github.com/spf13/afero v1.9.2
go: downloading github.com/spf13/jwalterweatherman v1.1.0
go: downloading github.com/bgentry/speakeasy v0.1.1-0.20220910012023-760eaf8b6816
go: downloading github.com/mattn/go-isatty v0.0.16
go: downloading github.com/cosmos/ledger-cosmos-go v0.12.2
go: downloading github.com/prometheus/client_golang v1.14.0
go: downloading github.com/prometheus/common v0.37.0
go: downloading golang.org/x/net v0.5.0
go: downloading github.com/davecgh/go-spew v1.1.1
go: downloading github.com/cosmos/iavl v0.19.5
go: downloading github.com/go-kit/log v0.2.1
go: downloading github.com/go-logfmt/logfmt v0.5.1
go: downloading github.com/gorilla/websocket v1.5.0
go: downloading github.com/improbable-eng/grpc-web v0.15.0
go: downloading github.com/coinbase/rosetta-sdk-go v0.7.9
go: downloading github.com/go-kit/kit v0.12.0
go: downloading github.com/lib/pq v1.10.6
go: downloading github.com/rs/cors v1.8.2
go: downloading github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d
go: downloading github.com/subosito/gotenv v1.4.1
go: downloading github.com/felixge/httpsnoop v1.0.2
go: downloading github.com/hashicorp/hcl v1.0.0
go: downloading gopkg.in/ini.v1 v1.67.0
go: downloading github.com/magiconair/properties v1.8.6
go: downloading github.com/pelletier/go-toml/v2 v2.0.5
go: downloading gopkg.in/yaml.v3 v3.0.1
go: downloading github.com/pmezard/go-difflib v1.0.0
go: downloading github.com/dvsekhvalnov/jose2go v1.5.0
go: downloading github.com/godbus/dbus v0.0.0-20190726142602-4481cbc300e2
go: downloading github.com/gsterjov/go-libsecret v0.0.0-20161001094733-a6f4afe4910c
go: downloading github.com/mtibben/percent v0.2.1
go: downloading golang.org/x/sys v0.4.0
go: downloading golang.org/x/term v0.4.0
go: downloading github.com/hashicorp/go-immutable-radix v1.3.1
go: downloading filippo.io/edwards25519 v1.0.0-rc.1
go: downloading github.com/btcsuite/btcd/btcec/v2 v2.3.2
go: downloading github.com/pelletier/go-toml v1.9.5
go: downloading golang.org/x/text v0.6.0
go: downloading github.com/zondax/ledger-go v0.14.1
go: downloading github.com/gtank/merlin v0.1.1
go: downloading github.com/libp2p/go-buffer-pool v0.1.0
go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369
go: downloading github.com/prometheus/client_model v0.3.0
go: downloading github.com/Workiva/go-datastructures v1.0.53
go: downloading github.com/rcrowley/go-metrics v0.0.0-20201227073835-cf1acfcdf475
go: downloading github.com/creachadair/taskgroup v0.3.2
go: downloading github.com/minio/highwayhash v1.0.2
go: downloading github.com/google/orderedcode v0.0.1
go: downloading github.com/beorn7/perks v1.0.1
go: downloading github.com/cespare/xxhash/v2 v2.1.2
go: downloading github.com/prometheus/procfs v0.8.0
go: downloading github.com/gtank/ristretto255 v0.1.2
go: downloading github.com/cenkalti/backoff/v4 v4.1.3
go: downloading github.com/desertbit/timer v0.0.0-20180107155436-c41aec40b27f
go: downloading nhooyr.io/websocket v1.8.6
go: downloading github.com/golang/snappy v0.0.4
go: downloading github.com/tidwall/btree v1.5.0
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/decred/dcrd/dcrec/secp256k1/v4 v4.0.1
go: downloading github.com/mimoo/StrobeGo v0.0.0-20210601165009-122bf33a46e0
go: downloading github.com/zondax/hid v0.9.1
go: downloading github.com/klauspost/compress v1.15.11
fc92d3b64b0618238b354bd82b31ed4aa35557129d679d90b070643df34c7c38  /root/go/bin/nobled
```

Comment (boojamya) [reply mainnet/noble-1/README.md:58]: Ah Ok.
Disregard then. I was getting it from: 
https://github.com/strangelove-ventures/noble/pkgs/container/noble/78874895?tag=v1.0.0

**Review by boojamya — APPROVED** [review]

**Pull request merged**